code/game/machinery/buttons.dm

```diff
@@ -131,6 +185,35 @@
 	active = 0
 	update_icon()
 
+/obj/machinery/button/mass_driver/attackby(obj/item/weapon/W, mob/user)

```

Comment:
This code looks an awful lot like `/obj/machinery/button/attackby(obj/item/weapon/W, mob/user)`, are there differences and can those be solved by referring to parent?